ivy_tests/test_ivy/test_functional/test_core/test_dtype.py

```diff
@@ -91,9 +136,18 @@ def test_broadcast_to(
     instance_method,
     fw,
 ):
-    x = data.draw(helpers.nph.arrays(shape=array_shape, dtype=input_dtype))
+    # smoke for torch uints
+    if fw == "torch" and "u" in str(dtype):
+        return
+    x = data.draw(helpers.nph.arrays(shape=in_shape, dtype=dtype))
+    to_shape = data.draw(
```

Comment:
we can pretty much use the same logic as you have written here, but refactor it into a private helper function (which we can store in the file `test_dtype.py` if you don't think it is general enough to add into `helpers.py`). The first argument of any `st.composite` function is `draw`, meaning we can wrap this exact same logic into a private helper function, which is then called inside the `@given` decorator. Feel free to commit a failing attempt if you can't get it working, and I can then take a look as well :)